src/components/StarChartViewer.vue

```diff
@@ -286,22 +303,56 @@ export default defineComponent({
       toast.succeed("CSV Downloaded");
     };
 
+    const handleShareToTwitterBtnClick = async () => {
+      const repos = store.state.repos;
+      if (repos.length === 0) {
+        toast.error("No repo found");
+        return;
+      }
+
+      const starhistoryLink = encodeURIComponent(window.location.href);
+      let text = "";
+
+      if (repos.length === 1) {
+        const repo = repos[0];
+        let starCount = 99;
```

Comment:
I feel, if we somehow don't get the count. We just use 0 and don't show the star count text at all